JitsiMeetJS.js

```diff
@@ -1,3 +1,4 @@
+require("babel-polyfill");
```

Comment:
Maybe add a check if `global._babelPolyfill` is already defined? Requiring `babel-polyfill` multiple times throws an error and it is possible that some other dependency (outside of lib-jitsi-meet) is also doing this.